scripts/Tools/case.build

```diff
@@ -132,6 +132,8 @@ def _main_func(description):
 
     success = True
     with Case(caseroot, read_only=False) as case:
+        case.record_cmd()
```

Comment:
I think I see here that everyplace where you call record_cmd() is directly after a case initialization.   I would suggest that you put this call inside the case class init and add an additional optional argument to that: record=False by default and if set to True calls the record_cmd function.

Reply:
@jasonb5 , why not add it to the `with` implementation (`__enter__`)?